loader/src/sources/albertsons/index.js

```diff
@@ -480,6 +488,82 @@ function formatLocation(data, validAt, checkedAt) {
     return null;
   }
 
+  const pharmacyMatch = findKnownAlbertsons(
+    `${address.lines[0]}, ${address.city}, ${address.state} ${address.zip}`,
+    { lat: data.lat, long: data.long },
+    storeBrand,
+    storeNumber
+  );
+
+  if (process.env.DEBUG) {
+    const matchType = pharmacyMatch?.method ?? "no_match";
+    knownLocationMatches[matchType] =
+      (knownLocationMatches[matchType] || 0) + 1;
+
+    if (pharmacyMatch) {
+      if (pharmacyMatch.data.c_parentEntityID !== storeNumber) {
+        const appAddress = `${address.lines[0]}, ${address.city}, ${address.state} ${address.zip}`;
+        const matAddress = `${pharmacyMatch.data.address.line1}, ${pharmacyMatch.data.address.city}, ${pharmacyMatch.data.address.region} ${pharmacyMatch.data.address.postalCode}`;
+        console.error(
+          "Method:",
+          pharmacyMatch.method,
+          " / Score:",
+          pharmacyMatch.score || "-",
+          " / Factors:",
+          pharmacyMatch.factors || "-"
+        );
+        const found = pharmacyMatch.data;
+        console.error(
+          "Appointments:",
+          // `${storeBrand.name} #${storeNumber}                      /  ${appAddress}  /  ${data.address}`
+          `${storeBrand.name} #${storeNumber}`.padEnd(40),
+          `/  ${appAddress}  /  ${data.address}`
+        );
+        console.error(
+          "Match:       ",
+          `${found.name} #${found.c_parentEntityID}`.padEnd(40),
+          `/  ${matAddress}`,
+          `\n              (c_oldStoreID: ${found.c_oldStoreID})`
+        );
+        console.error("------------");
+      }
+    } else if (
+      !["albertsons_corporate", "community_clinic"].includes(storeBrand?.key)
+    ) {
+      console.error("NO MATCH:", data.address);
+      const appAddress = `${address.lines[0]}, ${address.city}, ${address.state} ${address.zip}`;
+      console.error(
+        `  Parsed: ${storeBrand?.name} #${storeNumber}, ${appAddress}`
+      );
+      console.error("------------");
+    }
+  }
```

Comment:
Mind moving this out to its own function?

Reply:
Ah yeah, I was debating between that and just removing it entirely (I was mainly keeping it in until things were ready to merge in case I needed to re-do any of the above analyses). Will make sure to do one of those.

Reply:
Kept it, and moved to a separate function.